calcom/cal.com
# feat(i18n): complete Simplified Chinese translations

## What does this PR do?



- Added missing translation items in Simplified Chinese
- Added the script `check-missing-translations` to quickly compare whether there are missing translation items in a specific language
  - If there are no specific locales, then it'll check all locales under `apps/web/public/static/locales` directory.
  - I hardcode the value of the template file to be "en/common.json".



## Requirement/Documentation



I see that some translation items are not displayed in Simplified Chinese
<img width="899" alt="Screenshot 2024-04-09 at 17 57 41" src="https://github.com/calcom/cal.com/assets/100353458/86872c79-fabc-4ce9-917b-fef9ebc5d2b5">

## Type of change



- New feature (non-breaking change which adds functionality)

## How should this be tested?



Example: 
1. Run `yarn check-missing-translations zh-CN zh-TW` under apps/web.
2. Or just run `yarn check-missing-translations` to check missing translations for all languages.

## Mandatory Tasks


## Checklist
